CHANGELOG.md

```diff
@@ -0,0 +1,56 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+
+## [unreleased]
+### Added
+- CHANGELOG.md
+- strict type declarations to all classes (including tests)
+- type hinting for method parameters and return types
+- contracts for Router and View classes
+- php7.3 to travis config
+
+### Changed
+- added support for reflection based dependency injection 
```

Comment:
Maybe should be moved to `Added`?

Both:
 - added support for reflection based dependency injection 
 - resolve controller actions via container